src/global/state/navigationState.tsx

```diff
@@ -0,0 +1,40 @@
+import { atom, selector } from "recoil";
+import { loggedState } from "../../authentification/state/userState";
+import { Path } from "../types/navigationType";
+
+export const pathsState = atom<Path[]>({
+  key: "pathState",
+  default: [
+    {
+      name: "Register",
+      path: "/auth/register",
+      userLoggedIn: false,
+    },
+    {
+      name: "Log in",
+      path: "/auth/log-in",
+      userLoggedIn: false,
+    },
+    {
+      name: "My saved configurations",
+      path: "/auth",
+      userLoggedIn: true,
+    },
+    {
+      name: "Logout",
+      path: "/",
+      userLoggedIn: true,
+    },
+  ],
+});
+
+export const pathSelector = selector<Path[]>({
```

Comment:
This is okay thinking, but lets say i manually write in the url /auth while im not logged in. 
You will need to have private and public routes, when the user is logged in or not.

Reply:
Added guarding for Authentication route in App.tsx.